Makefile.linux64

```diff
@@ -1,6 +1,7 @@
 include Makefile
 
 CC=g++
-COPTS=-fPIC -O2 -m64 -Wall
-LDOPTS=-shared -Wl,-soname,$(LIBRARY)
+#CXXFLAGS=-std=c++11
+COPTS=-fPIC -O2 -m64 -Wall -std=c++11 -c
```

Comment:
There shouldn't be anything in this library that requires C++11 support

Reply:
c++11 is now required by MSP, and some new features we are looking to add (supporting multiple correlation functions) want to utilized std::shared_ptr, so this will be necessary going forward.